lms/djangoapps/certificates/views/webview.py

```diff
@@ -549,6 +557,13 @@ def render_html_view(request, user_id, course_id):
     # Get data from Discovery service that will be necessary for rendering this Certificate.
     catalog_data = _get_catalog_data_for_course(course_key)
 
+    # Determine the certificate language
+    course_title = active_configuration.get('course_title', '')
+    course_title = course_title if course_title else course.display_name
+    language = 'ar' if contains_rtl_text(course_title) else 'en'
+    translation.activate(language)
```

Comment:
```suggestion
```

This is very problematic and shouldn't be used. If you'd like to override the language the following form should be used:

```python
with translation.override(certificate_language):
   # something
```
override the [`certificate_language`](https://github.com/edx/edx-platform/blob/20426c761d77fcaeccc7f105ae5d794083095176/lms/djangoapps/certificates/views/webview.py#L546-L550) variable below instead.

I'll issue a follow up PR now.

Reply:
https://github.com/Edraak/edraak-platform/pull/137

Reply:
Thank you for help in this PR @OmarEdraak